docs/examples/edit_distance.md

```diff
@@ -0,0 +1,121 @@
+# Edit Distance
+
+- Given two strings `str1` and `str2`, what is the minimum number of edits
+  to convert `str1` to `str2`?
+- Each edit costs 1, and is either an add, delete, or replace.
+- Goal: find a minimum cost set of edits that converts `str1` to `str2`.
```

Comment:
Maybe for now just remove the concept of "cost". And instead just focus on "minimize the number of edits". As we've discussed earlier, we would like to put the extensions of edit distance (e.g., 1d array and alpha beta costs) into the blog. I want this to be something that a student can read in isolation of David's lecture on edit distance and understand fully.

Reply:
I think big David wanted something else, so we ended up going for that.